### ISSUE TITLE
NME 

### ISSUE THREAD

**Issue by kevin135106** [G-U-N/PyCIL#48]
Excuse me, I would like to ask the NME classifier on iCaRL, will it use the concept of knowledge distillation to calculate the loss? Or is it only available in the Softmax classifier (CNN)?

**Comment by zhoudw-zdw** [comment]
The training protocols are the same. The only difference between CNN acc and NME acc is the way you infer the label.
